wallet-new/src/Cardano/Wallet/API/V1/Types.hs

```diff
@@ -272,7 +273,32 @@ instance Arbitrary Payment where
                       <*> arbitrary
                       <*> arbitrary
 
-type TxId = Text
+----------------------------------------------------------------------------
+-- TxId
+----------------------------------------------------------------------------
+
+-- | TxId
+newtype TxId = TxId Text
+    deriving (Show, Eq, Generic)
+
+deriveJSON Serokell.defaultOptions ''TxId
+
+instance Arbitrary TxId where
+  arbitrary = TxId . fromString <$> elements
+      [ "1f434ae9e903ea86f420cd18160d2a6c4d5efa29a1004dfb0466f7a2ec643a6d"
```

Comment:
hm, I believe we have a better way of representing txId.

@Martoon-00 do we know in what format transaction id is? (ie, 32byte base58 string)

Reply:
I did this based on the example here - https://github.com/input-output-hk/cardano-sl/blob/ea906bd3a38ca560e7b13ddd6c0819fb532e15bd/wallet-new/src/Cardano/Wallet/API/V1/Types.hs#L107

Reply:
yes, that example was just a stub. But we should properly do txId arbitrary instance